funcs/func_frame_trace.c

```diff
@@ -379,6 +379,9 @@ static void print_frame(struct ast_frame *frame)
 		case AST_CONTROL_RECORD_MUTE:
 			ast_verbose("SubClass: RECORD_MUTE\n");
 			break;
+		case AST_CONTROL_PLAYBACK_BEGIN:
```

Comment:
Maybe keep the order of the "case" statements in line with the enum order (i.e. follow frame.h and move between AST_CONTROL_STREAM_FORWARD and AST_CONTROL_RECORD_CANCEL)?

Reply:
Yes, this switch does appear to be going in order so I'll go ahead and fix this, thanks!